sdks/how-to/how-to-create-a-data-sample-with-hierarchical-information.md

```diff
@@ -0,0 +1,393 @@
+---
+slug: how-to-create-a-data-sample-with-hierarchical-information
+description: Learn how to create a data sample with hierarchical information.
+tags:
+- DataSample
+
+---
+
+# How to create a data sample with hierarchical information
+
+In this section, we will learn how to create data samples with complex nested objects. DataSamples are used to store
+data that is not part of the patient's medical record. In some circumstances, the data sample may contain complex nested
+objects.
+
+For example, a data sample may contain another DataSample object with the blood pressure measurements, another
+DataSample object with the heart rate measurements, and so on.
+
+:::note
+
+We assume that you already know How to manage data samples. please follow
+the [Handling DataSamples](/sdks/how-to/how-to-manage-datasamples) guide
+
+:::
+
+## Create a DataSample object
```

Comment:
I think the organisation of this how-to is in general a bit confusing, I think it would be better to separate the time series example from the hierarchical/composite example.
Suggested new structure:
**Handling advanced data samples** (or similar)
- Hierarchical data samples
  - Create two data samples which may be nice to group
  - Put them in a hierarchical data sample
  - Tips/notes on:
    - You can only search/filter by parent, not by child
    - If you want to modify a child you will have to modify it from the parent
    - You can nest data samples in nested data samples
- Time series example
  - Example with a time series
  - Tip: you can also put time series data sample as children of hierarchical samples